sycl/test/usm/depends_on.cpp

```diff
@@ -39,7 +39,9 @@ int main() {
     return -1;
   }
 
+  event e;
```

Comment:
Commit message says that you are simplifying check, but as far as I understand you are fixing functional bug. If so, please update commit message with this information.